ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-adaptive-virtual-topology.cfg

```
@@ -9,6 +9,41 @@ router adaptive-virtual-topology
    region North_America id 1
    zone Canada id 2
    site Ottawa id 99
+   !
+   profile office365
+   !
+   profile scavenger
+      path-selection load-balance scavenger-lb
+      internet-exit policy scavenger-ie
+   !
+   profile video
+      internet-exit policy video-ie
+   !
+   profile voice
+      path-selection load-balance voice-lb
+   !
+   policy production
+      match application-profile videoApps
+      match application-profile criticalApps
+         profile crit
```

This still does not work
```suggestion
         avt profile crit
```